### DISCUSSION TITLE
Errors when modularizing domEvents

### DISCUSSION THREAD

**Discussion by Gabrielle-Tobermann** [nss-evening-cohort-14/evening-client-side#107] [Help Me!] [answered]
Name: Gabby Tobermann 

Searched other help tickets? Yes 

Issue: I was able to create modules for all my functions except for domEvents. In my domEvents file I put at the top: 
import { addStudentToArray, expelStudent } from './helpers/data/studentData';
import showForm from './components/showForm';

And at the bottom of my file I put: export default domEvents. 

In my JS file, I included 'import domEvents from './helpers/domEvents';'
I am getting a few errors that I cannot seem to fix. 

Errors: 
 1:49  error  Unable to resolve path to module './helpers/data/studentData'  import/no-unresolved
  1:49  error  Missing file extension for "./helpers/data/studentData"        import/extensions
  2:22  error  Unable to resolve path to module './components/showForm'       import/no-unresolved
  2:22  error  Missing file extension for "./components/showForm"             import/extensions

✖ 4 problems (4 errors, 0 warnings)

 @ ./src/javascripts/main.js 2:0-44 7:2-11

ERROR in ./src/javascripts/helpers/domEvents.js
Module not found: Error: Can't resolve './components/showForm' in '/home/gabrielle_tobermann/workspace/foundations/inclass/sorting-hat-webpack/src/javascripts/helpers'
 @ ./src/javascripts/helpers/domEvents.js 2:0-45 5:69-77
 @ ./src/javascripts/main.js

ERROR in ./src/javascripts/helpers/domEvents.js
Module not found: Error: Can't resolve './helpers/data/studentData' in '/home/gabrielle_tobermann/workspace/foundations/inclass/sorting-hat-webpack/src/javascripts/helpers'
 @ ./src/javascripts/helpers/domEvents.js 1:0-77 6:68-85 7:80-92
 @ ./src/javascripts/main.js
ℹ ｢wdm｣: Failed to compile.

Google links: https://stackoverflow.com/questions/47557583/how-to-fix-this-eslint-path-not-resolved-error-no-unresolved
https://github.com/microsoft/vscode-eslint/issues/595
https://github.com/houndci/hound/issues/1419

What I've tried: I haven't really tried anything because the solutions from the google links involve messing with the esLint config file. I have gone over my import and export statements but could not find what was wrong with them. '

Links to files: 
https://github.com/Gabrielle-Tobermann/sorting-hat-webpack/blob/modularization/src/javascripts/main.js
https://github.com/Gabrielle-Tobermann/sorting-hat-webpack/blob/modularization/src/javascripts/helpers/domEvents.js
https://github.com/Gabrielle-Tobermann/sorting-hat-webpack/blob/modularization/src/javascripts/helpers/data/studentData.js
https://github.com/Gabrielle-Tobermann/sorting-hat-webpack/blob/modularization/src/javascripts/components/showForm.js

**Comment by drteresavasquez** [comment]
Check your file paths to make sure they are correct. That is what the error is telling you. Also, I need links to code. Make sure to complete the whole ticket to get help you need.

**Reply by Gabrielle-Tobermann** [reply]
My bad! I added the links in the ticket.

**Comment by Gabrielle-Tobermann** [comment, marked as answer]
Update: my file paths in my import statements were incorrect. When you start typing the path "./blah/blah" your editor will tell you which folders you can access. It does all the work for you! If you hover over the path and nothing shows up, that means your path is incorrect.
